src/platform/audio/ffmpegaudiofilecache.cpp

```diff
@@ -0,0 +1,356 @@
+/**
+ * @file
+ *
+ * @author feliwir
+ *
+ * @brief Class for caching loaded audio samples to reduce file IO.
+ *
+ * @copyright Thyme is free software: you can redistribute it and/or
+ *            modify it under the terms of the GNU General Public License
+ *            as published by the Free Software Foundation, either version
+ *            2 of the License, or (at your option) any later version.
+ *            A full copy of the GNU General Public License can be found in
+ *            LICENSE
+ */
+
+extern "C" {
+#include <libavcodec/avcodec.h>
+#include <libavformat/avformat.h>
+}
+
+#include "audioeventrts.h"
+#include "audiomanager.h"
+#include "ffmpegaudiofilecache.h"
+#include "filesystem.h"
+#include <captainslog.h>
+#include <list>
+
+struct WavHeader
+{
+    /* RIFF Chunk Descriptor */
+    uint8_t RIFF[4] = { 'R', 'I', 'F', 'F' }; // RIFF Header Magic header
```

Comment:
Member name style should conform to project standards, normally `member_name` for public access structs. At the very least I'd like it consistent within a struct.

Reply:
fixed